build-profiling-ffi.sh

```diff
@@ -224,15 +223,25 @@ echo "Building tools"
 DESTDIR=$destdir cargo build --package tools --bins
 
 echo "Generating $destdir/include/libdatadog headers..."
-# ADD headers based on selected features.
-HEADERS="$destdir/include/datadog/common.h $destdir/include/datadog/profiling.h $destdir/include/datadog/telemetry.h $destdir/include/datadog/crashtracker.h"
+rm -r $destdir/include/datadog/
+mkdir $destdir/include/datadog/
+
+HEADER_FILES="common.h profiling.h telemetry.h crashtracker.h"
 case $ARG_FEATURES in
     *data-pipeline-ffi*)
-        HEADERS="$HEADERS $destdir/include/datadog/data-pipeline.h"
+        HEADERS="$HEADERS data-pipeline.h"
         ;;
 esac
+DEST_HEADERS=""
+for header in $HEADER_FILES; do
+    cp "$CARGO_TARGET_DIR/include/datadog/$header" "$destdir/include/datadog/$header"
+    DEST_HEADERS="$DEST_HEADERS $destdir/include/datadog/$header"
+done
+# Copy the blazesym header separately because they don't need to be duplicated
```

Comment:
deduplicated?

Reply:
Sorry, this is not clear. The blazesym header isn't auto-generated by cbindgen so we don't need to move remove definitions that are already present in `common.h`

Reply:
Fixed the comment